### PR TITLE
DeCo'd the original code which is no longer needed.

### PR DESCRIPTION
**Pull request by lvallone3315**
Please review my changes

### CODE REVIEW

File: ClassEnrollment/ClassEnrollment.cpp [hunk 1]
```diff
@@ -133,105 +88,3 @@ int main()
     studentDb.displayStudents(console, "Vector & List: after entering student IDs");
```
Comment (lvallone3315) [inline ClassEnrollment/ClassEnrollment.cpp:88]: Optional - you can delete line of code as well

File: ClassEnrollment/ClassEnrollment.cpp [hunk 2]
```diff
@@ -133,105 +88,3 @@ int main()
...
```
Comment (KhaniReed) [inline ClassEnrollment/ClassEnrollment.cpp:89]: test comment

File: ClassEnrollment/ClassEnrollment.cpp [hunk 3]
```diff
@@ -27,22 +27,8 @@
...
-    // two versions
-    //   1st vector of objects
-    //   2nd vector of pointers to objects
-std::vector<Student> studentVector;
-std::vector<Student*> studentPointerVector;
-
-// list approach
-//
-std::list<Student> studentList; 
+
```
Comment (ghost) [inline ClassEnrollment/ClassEnrollment.cpp:32]: This is in fact.... the main

**Review by ghost — APPROVED** [review]

**Review by KhaniReed — APPROVED** [review]

File: ClassEnrollment/ClassEnrollment.cpp [hunk 4]
```diff
@@ -55,46 +41,15 @@ int main()
...
-    sandboxStudentDb(console);
-
     // Parser approach to data entry
     //    prompt user for a command and while that command isn't quit, grab student IDs
     //    Generally - methods return true (action completed successfully) or false - action not completed
     //       ToDo - track type of failures & store in the parser output structure
     //       ToDo - store actual user string in the parser output structure (this ToDo is also captured elsewhere)
 
-    console.writeOutput("\nNext section - parser (separate class) to support user command entry\n");
-    system("pause");
```
Comment (HaileeHallard) [inline ClassEnrollment/ClassEnrollment.cpp:96]: You can delete this line of code

**Review by HaileeHallard — APPROVED** [review]

**Pull request merged**